src/index.ts

```diff
@@ -150,6 +151,145 @@ app.get("/api/users/:id", async (req, res) => {
   }
 });
 
+
+/**
+ * @openapi
+ * /api/users: 
+ *   delete:
+ *     summary: Removes user
+ *     responses:
+ *       200:
+ *         description: Successfully removed a user
+ *         content:
+ *           text/plain:
+ *             schema:
+ *               type: string
+ *       400:
+ *         description: Id has wrong data type
+ *         content:
+ *           text/plain:
+ *             schema:
+ *               type: string
+ *       404:
+ *         description: No user with given id
+ *         content:
+ *           text/plain:
+ *             schema:
+ *               type: string
+ *   parameters:
+ *   - name: id
+ *     in: path
+ *     description: Id of the user to be removed 
+ *     required: true
+*/
+
+app.delete("/api/users/:id", async (req, res) => {
+  const { id } = req.params;
+
+  if (typeof id !== "string") {
+    res
+      .status(400)
+      .send("Id has wrong data type");
+  }
+
+  try {
+    const user = await prisma.user.delete({
+      where: {
+        id: Number(id),
+      },
+    });
+
+    res.send(`User with id ${id} was deleted`);
+  } catch (e) {
+    res.status(404).send(`There is no user with id ${id}`);
+
+  }
+});
+
+/**
+ * @openapi
+ * /api/users: 
+ *   put:
```

Comment:
The exercise was to do a patch endpoint 😄 But I'll review this PR like it should've been a PUT instead, don't worry 😄